TicTacToe/TicTacToe.js

```diff
@@ -0,0 +1,157 @@
+class Game {
+    moves = 0;
+    user = 'X';
+    board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+    messages = "Game - START!";
+    complete = false;
+
+    reset() {
+        this.moves = 0;
+        this.user = 'X';
+        this.board = [['-', '-', '-'], ['-', '-', '-'], ['-', '-', '-']];
+        this.messages = "Game - START!";
+        this.complete = false;
+    }
+
+    getMoves() {
+        return this.moves;
+    };
+
+    displayBoard() {
+        let text = '';
+        for (let i = 0; i < 3; i++) {
+            for (let k = 0; k < 3; k++) {
+                text += " " + this.board[i][k];
+            }
+            text += "\n";
+        }
+        console.log(text);
+        return text;
+    };
+
+    getMessages() {
+        return this.messages;
+    }
+
+    makeMove(x, y) {
+        this.messages = "";
+        if (this.complete) {
+            this.messages = "GAME COMPLETED";
+            return;
+        }
+
+        if (this.board[x][y] != '-') {
```

Comment:
In JS when testing equality use `!==` and `===`. This checks both value and type equality.

(And/or consider typescript?)